src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,45 @@
 import React from 'react';
+import { FullPost } from '../../Types/FullPost';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+
+type Props = {
+  post: FullPost;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        {user && (
+          <p>
+            {`Posted by ${user.username}:`}
```

Comment:
I cannot load the demo link, but according to this line there seems to be two usernames in a row: from `${user.username}` and `<UserInfo user={user} />`. If I am right, you should remove `${user.username}`